src/main/scala/ch/openolitor/core/reporting/DocumentProcessor.scala

```diff
@@ -70,7 +70,7 @@ trait DocumentProcessor extends LazyLogging {
   val dateFormatter = ISODateTimeFormat.dateTime
   val libreOfficeDateFormat: DateTimeFormatter = DateTimeFormat.forPattern("dd.MM.yyyy HH:mm:ss")
 
-  val unaryFunctionsPattern: Regex = """\$(sum|avg|min|max|count)\("?([^"]+)"?\)""".r
+  val unaryFunctionsPattern: Regex = """\$(sum|avg|min|max|count|debug)\("?([^"]+)"?\)""".r
```

Comment:
Maybe debug could as well be a param1Function with the JsonPath as parameter to be able to debug the result of a JsonPath query?

Reply:
done